apps/desktop/e2e/tests/session-limit.spec.ts

```
@@ -0,0 +1,101 @@
+
+    // Open the temp project
+    const projectPath = path.join(fixture.tempDir, 'project');
+    await openProject(fixture.page, projectPath);
+  });
+
+  test.afterAll(async () => {
+    await closeApp(fixture);
+  });
+
+  // eslint-disable-next-line no-empty-pattern
+  test.afterEach(async ({}, testInfo) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)